nscplugin/src/main/scala/scala/scalanative/nscplugin/NirPrimitives.scala

```diff
@@ -128,7 +128,16 @@ abstract class NirPrimitives {
     addPrimitive(ULongToFloatMethod, ULONG_TO_FLOAT)
     addPrimitive(UIntToDoubleMethod, UINT_TO_DOUBLE)
     addPrimitive(ULongToDoubleMethod, ULONG_TO_DOUBLE)
-    HashMethods.foreach(addPrimitive(_, HASH))
+
+    {
+      import scala.tools.nsc.settings._
+      ScalaVersion.current match {
+        case SpecificScalaVersion(2, 11, _, _) =>
+          HashMethods.foreach(addPrimitive(_, HASH))
```

Comment:
I was not able to use the sbt-buildinfo plugin here to condition for the Scala version, because it resulted to an infinite recursion in sbt when resolving the projects that depend on `nscplugin` (tried to enable sbt-buildinfo for `nscplugin`).

I went with `nsc.settings.ScalaVersion` instead. I don't know if the latter has any version guarantees.